roles/install_ansible_rulebook/tasks/install_fedora.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Ensure system dependencies
+  become: true
+  ansible.builtin.dnf:
+    name:
+      - git
+      - python3
+      - python3-pip
+      - python3-devel
+      - java-17-openjdk-devel
+      - maven
+      - gcc
+
+- name: set global envvars
+  become: true
+  ansible.builtin.lineinfile:
+    path: /etc/environment
+    regexp: "{{ item.regexp }}"
+    line: "{{ item.line }}" 
+  loop:
+    - regexp: '^JAVA_HOME'
+      line: JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+    - regexp: '^RULES_ENGINE'
```

Comment:
This is no longer needed since drools is the default rules engine.

Reply:
fixed